index.html

```diff
@@ -66,47 +66,55 @@ <h2 class="intro">Haiku</h2>
         <section class="gallery">
         <h2>Photo Gallery</h2>
         <div class="gallery-thumbnails">
-        <figure class ="mugshot">
-        <a href="https://picsum.photos/id/10/1024">
-         <a class="row-span-2">
-        <img src="https://picsum.photos/id/10/200/200" alt="Placeholder Image" class="gray" />
+        <a href="https://picsum.photos/id/10/1024" class="row-span-2">
+            <img src="https://picsum.photos/id/10/300/600" alt="Placeholder Image" class="gray" />
```

Comment:
To add a class to an element, just add the class attribute to it.  An `<a>` may have both an href nad a class attribute.